docs/toolchain/testnet/testchain.md

```diff
@@ -3,7 +3,7 @@ id: testchain
 title: 测试链
 ---
 
-测试链的水龙头的地址为：https://faucet.citahub.com/faucet, 可以在这里领取 Testnet 的代币。
+测试链的水龙头的地址为: https://faucet.citahub.com/faucet/getNos, 可以在这里领取 Testnet 的代币。
```

Comment:
why updating the URL? https://faucet.citahub.com/faucet is working

Reply:
Yeah! I found it error in v1.0.0 doc.
Forgot this PR.